### ISSUE TITLE
GG Max Bug Report - The stab is not ignored by the CanHearSound system

### ISSUE THREAD

**Issue by UltraVox001** [TheGameCreators/GameGuruRepo#2138]
Even with a stab in the wind, enemies hear it and react. You need an option for all weapons because some weapons could be equipped with Silencers. Enemies should react to noises, but not Silencers or Knives.

In addition, when enemies hear a sound of gunfire, they move through walls at very high speed.

**Comment by UltraVox001** [comment]
@LeeBamberTGC 
You are right. I just tested with a Speed of 20. The enemy moved towards me in slow motion.
For silenced weapons (knives, pistol and silenced sniper), a new parameter is required. Without it, all these weapons are useless.

**Comment by rvanner** [comment]
@LeeBamberTGC In most games if you're heard then the enemies will come quick. They should not run through walls though! :-)

Knife attacks are generally accepted as quiet take downs and should not be part of the hearing system. Only if others see the attack should they get involved.

**Comment by LeeBamberTGC** [comment]
@rvanner Please post a 'character goes through wall' issue with step by step so I can reproduce. It should not be discussed in the stab sound issue.

**Comment by rvanner** [comment]
@LeeBamberTGC It wasn't me that reported that. It was @UltraVox001 who wrote _Also, when enemies hear a shooting sound, they will run through walls at very high speed._

**Comment by LeeBamberTGC** [comment]
@rvanner And I quote, "They should not run through walls though! :-)"

**Comment by UltraVox001** [comment]
@LeeBamberTGC @rvanner 
I guess this problem occurs because I'm on a full map that has "lost" the nav mesh. :)

**Comment by LeeBamberTGC** [comment]
@UltraVox001 Yes we have some nav mesh gotchas this week. Tick the AI management to see if you are standing on a nav mesh. Next weeks version 'hopefully' will have massive AND FAST nav meshes for all.
